README.md

```diff
@@ -1 +1,121 @@
-# python-SSL-checker
\ No newline at end of file
+## Python SSL checker
+
+---
+This project validate SSL certificate external site. If the certificate has expired then it is sent notification message on you Telegram bot otherwise the script shows the number of days before the expiration date.
+
+---
+
+### Requirements
+
+The code is written in Python 3.10.0.
+
+Before starting you must install the following Python-libraries:
+1. telegram_send
+```bash
+sudo pip3 install telegram-send
+```
+In the next step, you must create a telegram bot using @BotFather which will give you token to access the HTTP API looks like string 34334543534:DFTNMGFDdfddfadjkkmWfg.
+To connect telegram_send library to you Telegram bot you must run the following command in terminal:
+```bash
+telegram-send --configure
+```
+After that you must enter you Telegram token in terminal and press "Enter". The terminal code will appear in you Telegram bot.
+
+See more [Documentation](https://pythonhosted.org/telegram-send/#telegram-send)
+
+---
+### Description
+
+---
+The constant "DAYS_TO_EXPIRATION" contain number of days set manually. It is used to calculate the expiration date of the certificate.
+
+The boolean constant "SHOW" used as switch for send notification. By default is "TRUE".
+
+Function * *"def check_cert_validity()"* * used for validate SSL certificate on site. Required function argument is SSL certificate. Returned function value is "TRUE" or "FALSE" 
```

Comment:
Functions description is better to keep in code rather than in readme. It will be easier to read your code